examples/distributed_training.py

```diff
@@ -41,6 +41,14 @@ def compute_metrics(self, results):
         return dict(accuracy=100 * total_correct / total_size)
 
 
+class IntToTensor:
+    def __call__(self, data):
+        return torch.tensor(data)
+
+    def __repr__(self):
+        return self.__class__.__name__ + '()'
+
```

Comment:
This target_transform is not necessary here. Maybe you encountered error caused by MMEngine's default `collate_fn`? Our new [documentation](https://github.com/open-mmlab/mmengine/pull/696/files) (which is still WIP) detailed on this and you may give some feedback :smile: